docs/en/setup/backend/telegraf-receiver.md

```diff
@@ -0,0 +1,45 @@
+# Telegraf receiver
```

Comment:
Metrics receiver doc should be hosted in the menu like this

<img width="350" alt="image" src="https://user-images.githubusercontent.com/5441976/198949306-a7e2e327-4458-4d15-88ac-36fbe0b1ce14.png">

You should update the menu.yml in the docs folder.